runelite-client/src/main/java/net/runelite/client/plugins/coxhelper/CoxPlugin.java

```diff
@@ -80,9 +83,11 @@
 	private static final int GRAPHICSOBJECT_ID_HEAL = 1363;
 	private static final int ANIMATION_ID_G1 = 430;
 	private static final String OLM_HAND_CRIPPLE = "The Great Olm\'s left claw clenches to protect itself temporarily.";
-	private static final Pattern TP_REGEX = Pattern.compile("<col\\b[^>]*>(.*?)</col>");
+	private static final Pattern TP_REGEX = Pattern.compile("<col=ff0000>(.*)</col>");
+	private static final Pattern TEST_REGEX = Pattern.compile("You have been paired with <col=ff0000>(.*)</col>! The magical power will enact soon...");
```

Comment:
Is TEST_REGEX the best name for this pattern?

Reply:
Definitely not, thanks for pointing that out 👍